scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/protocol.kt

```diff
@@ -100,7 +101,9 @@ fun convertRequestToRawInputContents(request: V2Dataplane.ModelInferRequest): V2
                         .toList()
                 }.toByteArray()
             }
-            DataType.FP16, // as data is stored as FP32 we treat the same. Its unclear if Triton would handdle this correctly though
+            DataType.FP16 -> {  // as data is stored as FP32 we treat the same. Its unclear if Triton would handle this correctly though
```

Comment:
:upside_down_face: This comment doesn't make sense any more, as we're no longer treating this like FP32